src/main/java/world/bentobox/bentobox/api/commands/island/IslandSchemsPanelCommand.java

```diff
@@ -0,0 +1,40 @@
+package world.bentobox.bentobox.api.commands.island;
+
+import world.bentobox.bentobox.api.commands.CompositeCommand;
+import world.bentobox.bentobox.api.user.User;
+import world.bentobox.bentobox.panels.SchemsPanel;
+
+import java.util.List;
+
+/**
+ * @author barpec12
+ */
+public class IslandSchemsPanelCommand extends CompositeCommand {
```

Comment:
I don't see the point in creating this command. I'd rather like the SchemsPanel to be shown when executing `IslandCreateCommand`